vsts_cd_manager/continuous_delivery_manager.py

```diff
@@ -19,8 +19,7 @@
                                         CiArtifact, CiConfiguration, ProvisioningConfiguration,
                                         ProvisioningConfigurationSource, ProvisioningConfigurationTarget,
                                         SlotSwapConfiguration, SourceRepository, CreateOptions)
-from vsts_accounts import Account
-from vsts_accounts.models import (AccountCreateInfoInternal)
+from aex_accounts import Account
```

Comment:
IMO, name the new class as  account. and rename old ones as legacy_account or something similar

Reply:
Okay. But rename aex_accounts folder as just accounts and remove old files.